app-text/csvfix/metadata.xml

```diff
@@ -3,6 +3,6 @@
 <pkgmetadata>
   <!-- maintainer-needed -->
```

Comment:
Don't you want to maintain that?

Reply:
Not interested.
My original intention was tree-clean package, but after some feedback some devs pointed that main issues of package can be fixed. So why there this PR.
There still two options - delete package or let it remain in tree. Since no one even care about keeping it in tree, I prefer to delete.